src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -696,22 +696,32 @@ export class BaseEditCourseForm extends React.Component {
             courseSubmitting={submitting}
             {...this.props}
           />
+          {(!courseInReview && pristine &&
+          <Link to={`/courses/${uuid}/rerun`}>
+            <button
+              className="btn btn-block rounded mt-3 new-run-button"
+            >
+              <Icon className="fa fa-plus" /> Add Course Run
+            </button>
+          </Link>
+          )}
+          {(!pristine &&
+            <button
```

Comment:
This is also a lot of repeated code between these two states. Maybe you can make a variable like `disableAddRunButton` and set it equal to `!pristine` and then just have one button with `disabled={disableAddRunButton}`